package-coverage/generator/coverage.go

```diff
@@ -129,8 +129,11 @@ var execCoverage = func(dir string, coverageFilename string) error {
 		"-coverprofile=" + coverageFilename,
 	}
 
+	arguments = append(arguments, goTestArgs...)
+
 	cmd := exec.Command(command, arguments...)
 	cmd.Dir = dir
+	cmd.Stdout = os.Stdout
```

Comment:
Initially, we would forward `go test`'s stdout to our own if `-v` is specified. Not sure if that's a good idea, or if we should be doing some sanitisation of the output.